doc/news/_preparation_next_release.md

```diff
@@ -636,10 +336,7 @@ The website is generated from the repository, so all information about plugins,
 
 We are currently working on following topics:
 
-- <<TODO>>
 - Session recording and better Ansible integration _(Maximilian Irlinger @atmaxinger)_
```

Comment:
Copy from last release but remove your line:

```suggestion
- 1.0 API fixes _(Klemens Böswirth @kodebach)_ 
- Session recording and better Ansible integration _(Maximilian Irlinger @atmaxinger)_
- Change tracking _(Maximilian Irlinger @atmaxinger)_
- Rewriting tools in C _(@hannes99)_
- Elektrify KDE and GNOME _(Mihael Pranjić @mpranj)_
- Elektrify XFCE _(Richard Stöckl @Eiskasten)_
- Mounting SQL databases _(Florian Lindner @flo91)_
- Recording Configuration _(Maximilian Irlinger)_
- Ansible-Elektra _(Maximilian Irlinger)_
- Configure Olimex Base Images _(Maximilian Irlinger)_
- Improving Build Server Infrastructure _(Lukas Hartl)_
- Improve Java Development Experience _(Michael Tucek)_
```

Reply:
> - Change tracking _(Maximilian Irlinger @atmaxinger)_

This is already present in this release.

> - Session recording and better Ansible integration _(Maximilian Irlinger @atmaxinger)_

If you're splitting it up into separate points, than I think that should be removed.

Reply:
Copy from last release but remove your line:

```suggestion
- 1.0 API fixes _(Klemens Böswirth @kodebach)_ 
- Rewriting tools in C _(@hannes99)_
- Elektrify KDE and GNOME _(Mihael Pranjić @mpranj)_
- Elektrify XFCE _(Richard Stöckl @Eiskasten)_
- Mounting SQL databases _(Florian Lindner @flo91)_
- Recording Configuration _(Maximilian Irlinger)_
- Ansible-Elektra _(Maximilian Irlinger)_
- Configure Olimex Base Images _(Maximilian Irlinger)_
- Improving Build Server Infrastructure _(Lukas Hartl)_
- Improve Java Development Experience _(Michael Tucek)_
```

Reply:
Yes, I agree, thx for the fix.